playbooks/installation/installation.adoc

```diff
@@ -22,21 +23,53 @@ If adding those entries is problematic, a quick DNS solution for OpenShift is to
 
 === Node Sizing
 
-The Offcial Documentation lists some link:https://docs.openshift.com/enterprise/3.0/admin_guide/install/prerequisites.html#system-requirements[minimal system requirements] for OpenShift masters and nodes. Beyond that, there are several things to consider when deciding on the sizing for your nodes.
+|===============
+| Masters |
+Physical or virtual system, or an instance running on a public or private IaaS.
+
+Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic Host 7.2.4 or later.
+
+2 vCPU.
+
+Minimum 8 GB RAM.
+
+Minimum 30 GB hard disk space for the file system containing /var/.
+
+| Nodes |
+Physical or virtual system, or an instance running on a public or private IaaS.
+
+Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic Host 7.2.4 or later.
+
+NetworkManager 1.0 or later
+
+1 vCPU.
+
+Minimum 8 GB RAM.
+
+Minimum 15 GB hard disk space for the file system containing /var/.
+
+An additional minimum 15 GB unallocated space to be used for Docker’s storage back end; see Configuring Docker Storage below.
+|===============
 
 ==== Number of Nodes & Overall Environment Size
 
-When deciding how to size nodes, it's helpful to have an idea of the final size of your environment. This is important because knowing the total amount of resources your workloads will consume will help to dictate the amount of capacity loss your environment can handle. This is especially relevant in smaller environments, where the loss of one node could have significant impact on the performance and capacity of your overall environment. It's a good rule of thumb for environment sizing is to take the max expected total workload size and add 30% to that. This leaves about 10% for overhead on your nodes, plus 20% extra to account for node failure.
+When deciding how to size nodes, it’s helpful to have an idea of the final size of your environment. This is important because knowing the total amount of resources your workloads will consume will help to dictate the amount of capacity loss your environment can handle. This is especially relevant in smaller environments, where the loss of one node could have significant impact on the performance and capacity of your overall environment. It’s a good rule of thumb for environment sizing is to take the max expected total workload size and add 30% to that. This leaves about 10% for overhead on your nodes, plus 20% extra to account for node failure. The HA Smart Start Node configuration consists of 3 Masters, 2 Infrastructure Nodes, and 3 Application Nodes.
 
```

Comment:
add link to prerequisites sizing recommendations
https://docs.openshift.com/enterprise/3.2/install_config/install/prerequisites.html#sizing-recommendations